mailpoet/assets/js/src/landingpage/faq.tsx

```diff
@@ -0,0 +1,128 @@
+    readMoreLink: string;
+  }[] = [
+    {
+      slug: 'item-1',
+      title: __(
+        'What types of campaigns can I create with MailPoet?',
+        'mailpoet',
+      ),
+      text: __(
+        'MailPoet allows you to create five different types of emails: Newsletter, Welcome Email, Latest Post Notifications, Re-engagement Emails and WooCommerce.',
+        'mailpoet',
+      ),
+      readMoreText: __('Read More', 'mailpoet'),
+      readMoreLink:
+        'https://kb.mailpoet.com/article/141-create-an-email-types-of-campaigns',
+    },
+    {
+      slug: 'item-2',
+      title: __('How do I send a newsletter?', 'mailpoet'),
+      text: __(
+        'You can manually create a standard newsletter to be sent immediately or scheduled to be sent at a later time. Simply go to MailPoet > Emails and click on the “+ New Email” button to select “Newsletter”.',
+        'mailpoet',
+      ),
+      readMoreText: __('Read More', 'mailpoet'),
+      readMoreLink:
+        'https://kb.mailpoet.com/article/344-create-a-standard-newsletter',
+    },
+    {
+      slug: 'item-3',
+      title: __('Do I need a paid plan?', 'mailpoet'),
+      text: __(
+        "When you install the MailPoet plugin, you can use it for free up to 1,000 subscribers. If you have more than 1,000 subscribers, you'll need one of our paid plans: Creator, Business, or Agency. The best choice of plan type will depend on whether you want to send with our MailPoet Sending Service or your own sending method, as well as the number of sites you will be using MailPoet on.",
+        'mailpoet',
+      ),
+      readMoreText: __('Read More', 'mailpoet'),
+      readMoreLink:
+        'https://kb.mailpoet.com/article/349-choosing-your-mailpoet-plan',
+    },
+    {
+      slug: 'item-4',
+      title: __('How do I import my customers from WooCommerce?', 'mailpoet'),
+      text: __(
+        'The WooCommerce Customers list is a list automatically created by MailPoet with all of your WooCommerce customers. It also includes “Guest" customers. If WooCommerce is active, users that installed or updated the plugin should have chosen if they wanted to add the customers as “Subscribed” or “Unsubscribed” to the WooCommerce Customers list.',
+        'mailpoet',
+      ),
+      readMoreText: __('Read More', 'mailpoet'),
+      readMoreLink:
+        'https://kb.mailpoet.com/article/284-import-old-customers-to-the-woocommerce-customers-list',
+    },
+    {
+      slug: 'item-5',
+      title: __('How do I customize emails for my store?', 'mailpoet'),
+      text: __(
+        'You can create and send the following 4 WooCommerce Automatic emails with MailPoet: Abandoned Shopping Cart, First Purchase, Purchased In This Category, Purchased This Product. You can read more about each in our article.',
+        'mailpoet',
+      ),
+      readMoreText: __('Read More', 'mailpoet'),
+      readMoreLink:
+        'https://kb.mailpoet.com/article/277-woocommerce-automatic-emails',
```

Comment:
I think if these links came from constants we would be able to find and change them easier in future, what do you think?

Reply:
Hmmm 🤔, I'm not sure I agree with that.

Taking another look at the file, I think I'm okay with where the links are. 
It's super easy to understand what it refers to, which section and accordion item it describes and how to change it.

I believe it would be more confusing if we have multiple constants for the read-more links. We run into another stress of looking up the constants, checking the values, and understanding the context in which they were used.